source/Halibut/HalibutRuntime.cs

```diff
@@ -181,8 +194,14 @@ public TService CreateClient<TService>(ServiceEndPoint endpoint, CancellationTok
             return proxy;
 #endif
         }
-
+        
+        // TODO: async HalibutProxy
         ResponseMessage SendOutgoingRequest(RequestMessage request, CancellationToken cancellationToken)
+        {
+            return SendOutgoingRequestAsync(request, cancellationToken).GetAwaiter().GetResult();
```

Comment:
[Sync over async](https://github.com/davidfowl/AspNetCoreDiagnosticScenarios/blob/master/AsyncGuidance.md#warning-sync-over-async). Might make sense to implement a legit sync version of this method.

Reply:
One Day ™️ As discussed: an async HalibutProxy isn't within the scope of these changes. I've added a note for future us.